tensorflow/core/public/version.h

```diff
@@ -19,8 +19,8 @@ limitations under the License.
 // TensorFlow uses semantic versioning, see http://semver.org/.
 
 #define TF_MAJOR_VERSION 1
-#define TF_MINOR_VERSION 4
-#define TF_PATCH_VERSION 0
+#define TF_MINOR_VERSION 5
+#define TF_PATCH_VERSION 0rc0
```

Comment:
Isn't `TF_PATCH_VERSION` supposed to be 0 and `-rc0` should go in the `TF_VERSION_SUFFIX`?